command/sub8_missions/nodes/move_command

```diff
@@ -37,8 +47,49 @@ def main(args):
         fprint("Waiting for odom...")
         yield sub.tx_pose()
         fprint("Odom found!", msg_color='green')
-
+        rospack = rospkg.RosPack()
+        config_file = os.path.join(rospack.get_path('sub8_missions'), 'nodes', 'tp.yaml')
+        f = yaml.load(open(config_file, 'r'))
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
+        if args.gteleport:
+            try:
+                state_set_pub = yield nh.advertise('gazebo/set_model_state', ModelState)
+                try:
+                    x = float(args.gteleport.split(' ')[0])
+                    y = float(args.gteleport.split(' ')[1])
+                    z = float(args.gteleport.split(' ')[2])
+                except Exception:
+                    fprint("Format should be 'X Y Z' if you wish to input coords. Checking directory for saved location", msg_color='yellow')
+                    try:
+                        argz = f[args.gteleport]
```

Comment:
This will throw an vague key error exception if it isn't in the yaml. Instead you should always check that that key exsists
```
if args.teleport in f:
    # teleport
else:
   # notify user location not found
```